runtime/oti/cfr.h

```diff
@@ -958,6 +958,7 @@ typedef struct J9CfrClassFile {
 #if JAVA_SPEC_VERSION >= 16
 #define CFR_METHOD_EXT_HAS_SCOPED_ANNOTATION 0x08
 #endif /* JAVA_SPEC_VERSION >= 16 */
+#define CFR_METHOD_EXT_NOT_CHECKPOINT_SAFE_ANNOTATION 0x10
```

Comment:
```suggestion
#if defined(J9VM_OPT_CRIU_SUPPORT)
		#define CFR_METHOD_EXT_NOT_CHECKPOINT_SAFE_ANNOTATION 0x10
#endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
```

Reply:
this one is fine since we are not overloading that bit